.github/workflows/version-auto-bump.yml

```
@@ -0,0 +1,121 @@
+---
+name: Version Auto Bump
+
+on:
+  release:
+    types: [published]
+
+
+jobs:
+
+  setup:
+    name: "Setup"
+    runs-on: ubuntu-20.04
+    outputs:
+      version_number: ${{ steps.version.outputs.new-version }}
+    steps:
+      - name: Checkout Branch
+        uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579
+
+      - name: Get version to bump
+        id: version
+        run: |
+          CURR_MAJOR=$(echo ${{ github.event.release.tag }} | sed -r 's/v([0-9]{4}\.[0-9]\.)([0-9])/\1/'
+          CURR_VER=$(echo ${{ github.event.release.tag }} | sed -r 's/v([0-9]{4}\.[0-9]\.)([0-9])/\2/'
+          echo $CURR_VER
+
+          ((CURR_VER++))
+          NEW_VER=$CURR_MAJOR$CURR_VER
+
+          echo "::set-output name=new-version::$NEW_VER"
+
+  bump_version:
+    name: "Create version_bump branch"
+    runs-on: ubuntu-20.04
+    needs:
+      - setup
+    steps:
+      - name: Checkout Branch
+        uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579
+
+      - name: Create Version Branch
+        run: |
+          git switch -c version_bump_${{ needs.setup.outputs.version_number}}
```

```suggestion
        run: git switch -c version_bump_${{ needs.setup.outputs.version_number}}
```
This can be on a single line.